modules/openfoam/src/OpenFOAM_Registry.txt

```diff
@@ -11,8 +11,8 @@ include	Registry_NWTC_Library.txt
 
 
 # ..... OpenFOAM_InitInputType data .......................................................................................................
-typedef	OpenFOAM/OpFM	InitInputType	IntKi	NumActForcePtsBlade	-	-	-	"number of actuator line force points in blade"	-
-typedef	^	^	IntKi	NumActForcePtsTower	-	-	-	"number of actuator line force points in tower"	-
+typedef	OpenFOAM/OpFM	InitInputType	IntKi	NumActForcePtsBlade	-	-	-	"number of actuator line force points in blade -- from extern"	-
+typedef	^	^	IntKi	NumActForcePtsTower	-	-	-	"number of actuator line force points in tower -- from extern"	-
 typedef	^	^	ReKi	StructBldRNodes	{:}	-	-	"Radius to structural model analysis nodes relative to hub"
```

Comment:
Maybe we can mention in the description that these are obtained by linear interpolation of the AeroDyn nodes

Reply:
Eventually these will be values passed out to CFD to indicate how many points AD15 is providing through OpFM. But that is for phase ii of the updates when AD15 does the interpolation.  This of course will require API changes for AMR-Wind, NALU-Wind, and SOWFA...